core/config/identifiers.json

```diff
@@ -1,224 +1,20 @@
-      }
-    }
-  },
-  "CNY/USD": {
-    "dappConfig": {
-      "comment": "expiries: Sept 30, 2019 4:00PM EST, Oct 31st, 2019 4:00PM EST, Nov 29th, 2019 4:00PM EST, Dec 31st, 2019 4:00PM EST, Perpetual; supportedMove: 0.02 corresponds to 2%",
-      "expiries": [1569873600, 1572552000, 1575061200, 1577826000, 0],
+      "comment": "expiries: 6/1/2020 10am ET; 2% supported move",
+      "expiries": [1591020000],
       "supportedMove": "0.02"
     },
     "uploaderConfig": {
-      "publishInterval": "900",
+      "publishInterval": "0",
       "minDelay": "0",
       "numerator": {
         "dataSource": "Constant",
         "assetName": "1"
       },
       "denominator": {
-        "dataSource": "IntrinioForex",
-        "assetName": "USDCNH"
-      }
-    }
-  },
-  "Telegram SAFT": {
-    "dappConfig": {
-      "comment": "expiries: Sept 30, 2019 4:00PM EST, Oct 31st, 2019 4:00PM EST, Nov 29th, 2019 4:00PM EST, Dec 31st, 2019 4:00PM EST, Perpetual; supportedMove: 0.1 corresponds to 10%",
-      "expiries": [1569873600, 1572552000, 1575061200, 1577826000, 0],
-      "supportedMove": "0.1"
-    },
-    "uploaderConfig": {
-      "publishInterval": "900",
-      "minDelay": "0",
-      "numerator": {
-        "dataSource": "Constant",
-        "assetName": "100"
-      }
-    }
-  },
-  "USD/ETH": {
-    "dappConfig": {
-      "comment": "expiries: Sept 30, 2019 4:00PM EST, Oct 31st, 2019 4:00PM EST, Nov 29th, 2019 4:00PM EST, Dec 31st, 2019 4:00PM EST, Perpetual; supportedMove: 0.2 corresponds to 20%",
-      "expiries": [1569873600, 1572552000, 1575061200, 1577826000, 0],
-      "supportedMove": "0.2"
-    },
-    "uploaderConfig": {
-      "publishInterval": "900",
-      "minDelay": "0",
-      "numerator": {
-        "dataSource": "Constant",
-        "assetName": "1"
-      },
-      "denominator": {
-        "dataSource": "CMCCrypto",
-        "assetName": "ETH"
-      }
-    }
-  },
-  "Custom Index (1)": {
-    "dappConfig": {
-      "comment": "expiries: Sept 30, 2019 4:00PM EST, Oct 31st, 2019 4:00PM EST, Nov 29th, 2019 4:00PM EST, Dec 31st, 2019 4:00PM EST, Perpetual; supportedMove: 0.2 corresponds to 20%",
-      "expiries": [1569873600, 1572552000, 1575061200, 1577826000, 0],
-      "supportedMove": "0.2"
-    },
-    "uploaderConfig": {
-      "publishInterval": "900",
-      "minDelay": "0",
-      "numerator": {
-        "dataSource": "Constant",
-        "assetName": "1"
-      }
-    }
-  },
-  "Custom Index (100)": {
-    "dappConfig": {
-      "comment": "expiries: Sept 30, 2019 4:00PM EST, Oct 31st, 2019 4:00PM EST, Nov 29th, 2019 4:00PM EST, Dec 31st, 2019 4:00PM EST, Perpetual; supportedMove: 0.1 corresponds to 10%",
-      "expiries": [1569873600, 1572552000, 1575061200, 1577826000, 0],
-      "supportedMove": "0.2"
-    },
-    "uploaderConfig": {
-      "publishInterval": "900",
-      "minDelay": "0",
-      "numerator": {
         "dataSource": "Constant",
-        "assetName": "100"
+        "assetName": "102"
```

Comment:
Just out of curiosity, what's the 102 denominator?

Reply:
`1/102` was just the value of KES/USD at the time I deployed it, so I figured I'd just hardcode it in the config rather than building out any real price feed infrastructure.